zerver/openapi/zulip.yaml

```diff
@@ -2478,6 +2519,7 @@ paths:
   /server_settings:
     get:
       operationId: get_server_settings
+      tags: ["server_and_ organizations"]
```

Comment:
typo here -- we shoudln't have that space.

Reply:
fixed